modules/programs/just.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.just;
+
+in {
+  meta.maintainers = [ maintainers.maximsmol ];
+
+  options.programs.just = {
+    enable = mkEnableOption
+      "just is a handy way to save and run project-specific commands";
```

Comment:
This expands to `Whether to enable just is a handy way...`.
The phrasing seems odd.